src/Cache/Engine/NullEngine.php

```diff
@@ -113,4 +122,20 @@ public function clearGroup($group)
     {
         return false;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function get($key, $default = null)
+    {
+        return false;
```

Comment:
Shouldn't this return `$default`?

Reply:
Hm... whats the purpose of the Null engine? In which scenario do we want the default instead of false?